### ISSUE TITLE
Align with Infra

### ISSUE THREAD

**Issue by annevk** [whatwg/url#282]
- [ ] Reword any tuple language
- [ ] Rephrase URL records and hosts to be structs
- [ ] Investigate removing labels from the IPv6 parser
- ...

**Comment by domenic** [comment]
Eh, they seem like structs already; it seems like you'd just call them a struct and be pretty much done?

**Comment by annevk** [comment]
Yeah okay, I guess I'm wondering if that's something we want to do. Should all our concepts bottom out in these types? Should request and response be structs too?
